dnsapi/dns_hostingde.sh

```diff
@@ -38,6 +40,18 @@ _hostingde_apiKey() {
   _saveaccountconf_mutable HOSTINGDE_ENDPOINT "$HOSTINGDE_ENDPOINT"
 }
 
+_hostingde_parse() {
+  find="${1}"
+  if [ "${2}" ]; then
+    notfind="${2}"
+  fi
+  if [ "${notfind}" ]; then
+    _egrep_o \""${find}\":.*" | grep -v "${notfind}" | cut -d ':' -f 2 | cut -d ',' -f 1 | tr -d '[:space:]'
+  else
+    _egrep_o \""${find}\":.*" | cut -d ':' -f 2 | cut -d ',' -f 1 | tr -d '[:space:]'
```

Comment:
please do not use `'[:space:]'`, it's not supported crossing all the platforms.
please just use plain space char `' '` instead.

Thanks.

Reply:
Hi Neil,
I updated the tr command.